### PR TITLE
docs: add note about ngModel usage in structural directives guide

### PR DESCRIPTION
**Pull request by javatutorials2016**
When I tried one of the examples provided in the documention, encountered the " Can't bind to 'ngModel' since it isn't a known property of 'select' ” error. So to resolve it I imported the FormsModule in ngModule. If it is mentioned in the documentation, it would be handy for the beginners to try and learn. Thanks for considering.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by kapunahelewong — APPROVED** [review]
LGTM

File: aio/content/guide/structural-directives.md [hunk 1]
```diff
@@ -590,7 +590,7 @@ The drop down works properly.
   <img src='generated/images/guide/structural-directives/select-ngcontainer-anim.gif' alt="ngcontainer options work properly">
 </figure>
 
-
+<i>Note: You need to import the FormsModule package from @angular/forms in your Angular app.module.ts file to try the above example. ngModel is under FormsModule.</i>
```
Comment (brandonroberts) [inline aio/content/guide/structural-directives.md]:
```suggestion
<div class="alert is-important">

**Note:** You need to add the `FormsModule` to your `NgModule` imports array to use ngModel in the example above.

</div>

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by brandonroberts** [comment]
The git commit message needs to be updated to describe the change also. `docs: add note about ngModel usage in structural directives guide`

**Comment by brandonroberts** [comment]
@javatutorials2016 are you still interested in landing this change?

**Comment by javatutorials2016** [comment]
Hi Brandon, yes. Any action pending from my side? Please let me know thanks

**Review by javatutorials2016 — COMMENTED** [review]
docs: add note about ngModel usage in structural directives guide

**Comment by brandonroberts** [comment]
Something happened with the CLA. Will you rebase and squash so you're the only author?

**Comment by javatutorials2016** [comment]
Thanks. Will do as soon as I reach home . At office now thanks

**Review by javatutorials2016 — COMMENTED** [review]
docs: add note about ngModel usage in structural directives guide

**Comment by javatutorials2016** [comment]
Hi Brandon,
   Shall I create a new pull request for the same? I did the changes
directly on github portal so when I clone now, I don’t see my changes.
Kindly suggest and thanks for your time.
John

**Comment by brandonroberts** [comment]
@javatutorials2016 if that would be easier for you, then yes just create a new one.

**Comment by javatutorials2016** [comment]
thanks. Created a new one# 29522

**Pull request closed without merging**